cmd/mongodb/main.go

```diff
@@ -0,0 +1,126 @@
+package main
+
+import (
+	"fmt"
+	"net/http"
+	"os"
+	"os/signal"
+	"syscall"
+
+	kitprometheus "github.com/go-kit/kit/metrics/prometheus"
+	"github.com/mainflux/mainflux"
+	log "github.com/mainflux/mainflux/logger"
+	"github.com/mainflux/mainflux/writers"
+	mongodb "github.com/mainflux/mainflux/writers/mongodb"
+	nats "github.com/nats-io/go-nats"
+	stdprometheus "github.com/prometheus/client_golang/prometheus"
+)
+
+const (
+	name       = "mongodb-writer"
+	defNatsURL = nats.DefaultURL
+	defPort    = "8180"
+	defDBName  = "mainflux"
+	defDBHost  = "localhost"
+	defDBPort  = "8087"
+	defDBUser  = "mainflux"
+	defDBPass  = "mainflux"
+
+	envNatsURL = "MF_NATS_URL"
+	envPort    = "MF_MONGO_WRITER_PORT"
+	envDBName  = "MF_MONGO_WRITER_DB_NAME"
+	envDBHost  = "MF_MONGO_WRITER_DB_HOST"
+	envDBPort  = "MF_MONGO_WRITER_DB_PORT"
```

Comment:
Rename to `MF_MONGODB_DB_NAME`, `MF_MONGODB_DB_HOST`...

Reply:
@dusanb94 Dejan demanded change `MONGODB -> MONGO` and also change in InfluxDB to reflect this change. I agree whatever you guys decide, both look good for me, especially because InfluxDB is rarely referred to as just Influx.

Reply:
The reason I suggested these changes is database name. It's MongoDB, not jus Mongo. For the same reason, I suggested changes in mongoRepo => mongodbRepo.